challenge.rb

```diff
@@ -1,26 +1,26 @@
 
 def capitalize_each_string(input)
-  #implement your solution here
+input.map { |n| n.capitalize }
```

Comment:
looks good. take a look at the Symbol#to_proc method. it's very common.
https://ruby-doc.org/core-1.9.3/Symbol.html#method-i-to_proc